### DISCUSSION TITLE
Query parameters en `GET /tasks/group` no funcionan

### DISCUSSION THREAD

**Discussion by Mauriceac** [Cotalker/documentation#585] [4. Bug Reports]
El API request `GET /v2/tasks/group` no funciona. 
Sin embargo, `GET /v1/tasks/group` sí funciona, pero no sus query parameters.

Request Sample:
```bash
curl --location --request GET 'https://www.cotalker.com/api/v1/tasks/group?count=true&isActive=false&limit=1&page=2' \
--header 'Admin: true' \
--header 'Authorization: Bearer $ACCESS_TOKEN
```

**Comment by acedward** [comment]
/tasks no existe bajo /v2
solo bajo /v1 y /v3

/v1 y /v2 son rutas "reales" del sistema
/v3 junta ambas apis

**Reply by Mauriceac** [reply]
Ok. 
Tener presente:
- v3 no funciona.
- Swagger no da la opción de probar en v1.
- El problema real es que los query parameters no funcionan en `GET /tasks/group`
